internal/pkg/instrumentation/manager.go

```diff
@@ -207,7 +210,7 @@ func (m *Manager) cleanup(target *process.TargetDetails) error {
 func (m *Manager) Close() error {
 	m.done <- true
 	m.wg.Wait()
```

Comment:
There is no guarantee that that probes are finished as this line can be reached before `m.wg.Add(len(m.probes))` is called.

Shouldn't we call `m.wg.Wait()` after `err := <-m.runningErrors`?

Reply:
fixed